zeppelin-web/src/components/navbar/navbar.controller.js

```diff
@@ -115,4 +115,8 @@ angular.module('zeppelinWebApp').controller('NavCtrl', function($scope, $rootSco
   vm.loadNotes();
   $scope.checkUsername();
 
+  $rootScope.dateToString = function(date) {
```

Comment:
In that case it would be through a service, not the rootScope.
(The elapsed time function is a bit of a special case)

Overall since moment.js library call is already quite direct, there isn't really a need for it.